R/moduleInstall.R

```diff
@@ -1,12 +1,338 @@
+postInstallFixes <- function(folderToFix) {
+  if(length(ls(all.names=TRUE,pattern="\\..postProcessLibraryModule")) > 0 ) #We seem to be running in JASP
+  {
+    #print("we are *in* jasp, so we use .postProcessLibraryModule!")
+    .postProcessLibraryModule(folderToFix)
+  }
+  else 
+  {
+    #We do not have that function available so we will need to start JASPEngine ourselves, but where is it?
+    jaspEngineLocation <- ifElse(Sys.getenv("JASPENGINE_LOCATION") != "", Sys.getenv("JASPENGINE_LOCATION"), paste0(getwd(), "/../JASPEngine"))
```

Comment:
more ideomatic would be
```suggestion
    jaspEngineLocation <- Sys.getenv("JASPENGINE_LOCATION", unset = file.path(getwd(), "..", "JASPEngine"))
```
without that, it probably also doesn't work on Windows, although we probably don't need this there anyway.